addremove.test.js

```diff
@@ -3,7 +3,7 @@ import { items, addItem, removeItem } from './__mock__/index.js';
 describe('Adding and removing functions', () => {
```

Comment:
- [ ] 1 of your tests is failing kindly fix it

Reply:
- [ ] Remember to uses mock for functions that used localStorage, DOM, or external APIs and to add more tests